vision/google/cloud/vision/_gax.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX Client for interacting with the Google Cloud Vision API."""
+
+
+class _GAPICVisionAPI(object):
+    """Vision API for interacting with the gRPC version of Vision.
+
+    :type client: :class:`~google.cloud.core.client.Client`
+    :param client: Instance of ``Client`` object.
+    """
+    def __init__(self, client=None):
+        raise NotImplementedError
```

Comment:
This is OK to check in, but a do-nothing constructor would've been fine too (i.e. the `NotImplementedError` not needed as much for the constructor as for instance methods)